searches/interpolation_search.php

```diff
@@ -0,0 +1,23 @@
+function Search($list, $data) {
```

Comment:
Per the [PSR-12 php style guide](https://www.php-fig.org/psr/psr-12/), every PHP file should have an opening `<?php` tag